vllm/model_executor/models/qwen2_vl.py

```diff
@@ -775,62 +792,60 @@ def input_processor_for_qwen2_vl(ctx: InputContext,
         )["input_ids"]
 
     # Expand image pad tokens.
-    if image_inputs is not None:
-        image_indices = [
+    def expand_pad_tokens(inputs, token_id, make_batched_fn, data_type_key):
```

Comment:
Can you extract this function to the global scope to avoid conflicting variable names with the outer function scope?

Also, please add type annotations for each parameter.

Reply:
In fact, that's how I first implemented it, but image_processor, prompt_token_ids should be the same, and it would introduce too many parameters and impact the global scope.